lib/pages/chat/chat_pinned_events/pinned_events_view.dart

```diff
@@ -38,10 +38,6 @@ class PinnedEventsView extends StatelessWidget {
             switch (success.runtimeType) {
               case ChatGetPinnedEventsSuccess:
                 final data = success as ChatGetPinnedEventsSuccess;
-                if (data.pinnedEvents.isEmpty) {
```

Comment:
why is it not needed anymore

Reply:
It's not empty because in `ChatGetPinnedEventsInteractor` I checked if it was empty or not, if empty I returned `ChatGetPinnedEventsNoResult`